examples/angular/README.md

```diff
@@ -1,64 +1,153 @@
-cd core
-git sparse-checkout set --no-cone examples/angular
-git checkout
-```
-The files will be found under the `examples/angular` folder
+### Downloading the example
+
+You can get the code in two ways:
+
+1. Direct download:
+   ```
+   https://github.com/dotCMS/core/tree/master/examples/angular
+   ```
+
+2. Using Git sparse checkout:
+   ```bash
+   git clone -n --depth=1 --filter=tree:0 https://github.com/dotCMS/core
+   cd core
+   git sparse-checkout set --no-cone examples/angular
+   git checkout
+   ```
+   The example files will be in the `examples/angular` folder.
 
+### Configuration
 
+To configure the Angular app to use your dotCMS instance:
 
-## Add the dotCMS configuration
+1. Open the project folder in your code editor
+2. Navigate to `src/environments`
+3. Open `environment.development.ts` and update the following variables:
+   - `authToken`: Your dotCMS auth token
+   - `dotcmsUrl`: URL of your dotCMS instance (e.g., https://demo.dotcms.com)
 
-Now we need to tell the Angular app what dotCMS instance is going to use to get the data to build its pages.
+   ```typescript
+   export const environment = {
+     production: false,
+     authToken: 'YOUR_AUTH_TOKEN_HERE',
+     dotcmsUrl: 'https://demo.dotcms.com',
+     // other configurations...
+   };
+   ```
 
-1. Open the folder `YOUR_NAME` in your code editor
-2. Go to `src/environments`
-3. Open the `environment.development.ts` file and update the environment variable:
+   ⚠️ **Security Note**: Ensure that the `authToken` used here has read-only permissions to minimize security risks in client-side applications.
 
-- `authToken` this is the auth token for dotCMS, you can use the dotCMS UI to create one.
-- `dotcmsUrl` this is the instance of dotCMS where your pages and content lives (license needed) if you don’t have one, you can use [https://demo.dotcms.com](https://demo.dotcms.com) (be careful it restarts every 24h)
+## Running the Application
 
-## Run the app
+Once configured, follow these steps to run the app:
 
-Once all the configuration is in place, it is time to run the web app.
+1. Open a terminal in the project root directory
+2. Install dependencies: `npm install`
+3. Start the development server: `ng serve`
+4. Open your browser and navigate to `http://localhost:4200`
 
-1. Go back to your terminal and from the folder YOUR_NAME
-2. Run `ng serve`
-3. Open http://localhost:4200 in your browser
+🎉 Congratulations! Your dotCMS Angular example is now running.
 
-🎉 And that’s it.
+Note: When accessing `localhost:4200/about`, ensure that the `/about` page exists in your dotCMS instance.
+
+## Project Structure
+
+```
+.
+└── src/
+    └── app/
+        ├── content-types/
+        │   ├── activity/
+        │   ├── banner/
+        │   ├── product/
+        │   └── ...other-content-types/
+        ├── pages/
+        │   ├── components/
+        │   └── services/
+        └── shared/
+            ├── contentlets/
+            └── contentlet/
```

Comment:
These two folder name are too similar, tend to confuse users, we can either rename or change location, or join, or something.